app/src/main/kotlin/com/intive/picover/profile/viewmodel/ProfileViewModel.kt

```diff
@@ -35,7 +36,17 @@ class ProfileViewModel @Inject constructor(
 		}
 	}
 
+	fun updateAvatar(uri: Uri) {
+		viewModelScope.launch {
+			_profile.value = Loading
+			authRepository.updateUserAvatar(uri)
```

Comment:
what if this request fail ?

Reply:
Now I didn't handle it I can add it in the near PR for this task